### ISSUE TITLE
Soft hyphen (00AD) is invisible

### ISSUE THREAD

**Issue by drs-p** [clauseggers/Playfair-Display#68]
The soft hyphen in the Playfair fonts is invisible and has width zero. That is a problem in a Tagged PDF as soft hyphens should be used for hyphenated words to improve accessibility and screen reading. See the discussion at this issue: https://github.com/latex3/tagging-project/issues/967, and also https://jkorpela.fi/shy.html#pdf for a discussion about the soft hyphen in general and its use in PDF specifically.

**Comment by clauseggers** [comment]
Fixed. [db16f87](https://github.com/clauseggers/Playfair/commit/db16f876e5d71a39108ccb3707d6f77a2182d32e) closes.
